libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/components/MatrixUserRow.kt

```diff
@@ -46,11 +46,11 @@ fun MatrixUserRow(
     matrixUser: MatrixUser,
     modifier: Modifier = Modifier,
     avatarSize: AvatarSize = matrixUser.avatarData.size,
-    onClick: () -> Unit = {},
+    onClick: (() -> Unit)? = null,
 ) {
     Row(
         modifier = modifier
-            .clickable(onClick = onClick)
+            .let { if (onClick != null) it.clickable(onClick = onClick) else it }
```

Comment:
No sure but from my perspective, it is better to have a dedicated parameter "isClickable" than relying on the nullability

Reply:
by setting the lambda to `null`?

Reply:
I think we should keep the entire row clickable when there is a checkbox according to [Material guidelines](https://m3.material.io/components/checkbox/accessibility). I'll restore the default lambda, try to fix the issue with the ripple effect which is not applied to the checkbox, and update the checkbox when clicking on the row.

For me, it's not a problem to have an effect with no action. For example, in the `Android settings -> About`, there are some rows with an effect but no action.

Wdyt guys?

Reply:
I finally removed `clickable` by default, we'll have to add it manually now, not sure it's a big deal https://github.com/vector-im/element-x-android/pull/223/commits/2f4ce51a3618bcfcb57df04761bc942574fac41b